hack/lib/init.sh

```diff
@@ -62,4 +65,6 @@ github.com/openshift/api/template/v1,\
 github.com/openshift/api/user/v1\
 "
 
-TYPE_PACKAGE_VERSIONS="apiserver:v1 apps:v1 authorization:v1 build:v1 config:v1 helm:v1beta1 console:v1 console:v1alpha1 image:v1,docker10,dockerpre012 imageregistry:v1 kubecontrolplane:v1 legacyconfig:v1 cloudnetwork:v1 network:v1 networkoperator:v1 oauth:v1 openshiftcontrolplane:v1 operator:v1 operator:v1alpha1 operatorcontrolplane:v1alpha1 operatoringress:v1 osin:v1 project:v1 quota:v1 route:v1 samples:v1 security:v1 securityinternal:v1 servicecertsigner:v1alpha1 sharedresource:v1alpha1 template:v1 user:v1 machine:v1beta1 machine:v1 monitoring:v1alpha1"
\ No newline at end of file
+TYPE_PACKAGE_VERSIONS="apiserver:v1 apps:v1 authorization:v1 build:v1 config:v1 helm:v1beta1 console:v1 console:v1alpha1 image:v1,docker10,dockerpre012 imageregistry:v1 kubecontrolplane:v1 legacyconfig:v1 cloudnetwork:v1 network:v1 networkoperator:v1 oauth:v1 openshiftcontrolplane:v1 operator:v1 operator:v1alpha1 operatorcontrolplane:v1alpha1 operatoringress:v1 osin:v1 project:v1 quota:v1 route:v1 samples:v1 security:v1 securityinternal:v1 servicecertsigner:v1alpha1 sharedresource:v1alpha1 template:v1 user:v1 machine:v1beta1 machine:v1 monitoring:v1alpha1"
+
+OPENAPI_GEN="go run ${PACKAGE_NAME}/tools/vendor/k8s.io/code-generator/cmd/openapi-gen"
```

Comment:
Got it, this makes sense to me and would work temporarily.  Is this the part that doesn't work in CI?

Reply:
The part that doesn't work in CI is using Go workspaces, which was an attempt to avoid polluting the `go.mod` of the API module. If we only have one go module and run everything from the top level vendor it works, if we use workspaces, it doesn't work in CI as CI sets `-mod=vendor` which is incompatible.

The alternative is to build the tools from the vendor which avoid any of the headache I was trying to workaround last week. Going to push that shortly